src/Microsoft.Diagnostics.Monitoring.EventPipe/Counters/CounterFilter.cs

```diff
@@ -19,10 +21,10 @@ public static CounterFilter AllCounters(float counterIntervalSeconds)
         public CounterFilter(float intervalSeconds)
         {
             //Provider names are not case sensitive, but counter names are.
-            _enabledCounters = new Dictionary<string, List<string>>(StringComparer.OrdinalIgnoreCase);
+            _enabledCounters = new(StringComparer.OrdinalIgnoreCase);
 
             //The Series payload of the counter which we use for filtering
-            _intervalMilliseconds = (int)(intervalSeconds * 1000);
+            _intervalMilliseconds = ConvertInterval(intervalSeconds);
```

Comment:
Nit: This feels like unnecessary abstraction to me. If you do keep the method perhaps call it 'SecondsToMs()'